### PR TITLE
fix invalid link for reserved resource name

### PR DESCRIPTION
**Pull request by kerthcet**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
 None
```

### CODE REVIEW

File: site/content/en/docs/concepts/cluster_queue.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Kueue assigns the first flavor in the ClusterQueue's `.spec.resourceGroups[*].fl
 list that has enough unused `nominalQuota` quota in the ClusterQueue or the
 ClusterQueue's [cohort](#cohort).
 
-Since `pods` resource name is [reserved](/docs/conceps/workload#reserved-resource-names) and it's value
+Since `pods` resource name is [reserved](/docs/concepts/workload/#reserved-resource-names) and it's value
```
Comment (kerthcet) [inline site/content/en/docs/concepts/cluster_queue.md:66]: Here's the hole, other parts are formats.

Comment (tenzen-y) [reply site/content/en/docs/concepts/cluster_queue.md:66]: This is a nice catch.

File: site/content/en/docs/concepts/cluster_queue.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ small resource requesting workloads admitted at the same time.
 
 It is possible that multiple resources in a ClusterQueue have the same flavors.
 This is typical for `cpu` and `memory`, where the flavors are generally tied to
-a machine family or VM availability policies. To tie two or more resources to 
+a machine family or VM availability policies. To tie two or more resources to
```
Comment (tenzen-y) [inline site/content/en/docs/concepts/cluster_queue.md:75]: I think these spaces are intended.

Comment (kerthcet) [reply site/content/en/docs/concepts/cluster_queue.md:75]: Why? we have a newline here.

Comment (tenzen-y) [reply site/content/en/docs/concepts/cluster_queue.md:75]: Even If we have a new line, we used to add space like in most places. For example, you can see in Line 69.
If we want to remove these spaces, we need to work on all the places in the documentation.

Comment (kerthcet) [reply site/content/en/docs/concepts/cluster_queue.md:75]: > we used to add space like in most places

Do we? What I saw is most of the lines doesn't have the space.

Comment (tenzen-y) [reply site/content/en/docs/concepts/cluster_queue.md:75]: I checked spaces in https://raw.githubusercontent.com/kubernetes-sigs/kueue/91ee062d4e5f63dfb5b7b5aec5139c45a5156555/site/content/en/docs/concepts/cluster_queue.md.

And then, most places have a space. However, once I confirmed spaces in my local, as you said, there aren't spaces in most places.

It looks like the GitHub adds space to all places...

**Review by tenzen-y — COMMENTED** [review]
Thanks!
/lgtm
/approve

**Pull request merged**